docs/user/guides/search-analytics.rst

```diff
@@ -0,0 +1,39 @@
+How To Use Search Analytics
+===========================
+
+In this guide, you can learn to use Read the Docs' built-in search analytics for your documentation project.
+
+To see a list of the top queries and an overview from the last month,
+go to the :guilabel:`Admin` tab of your project,
+and then click on :guilabel:`Search Analytics`.
+
+.. figure:: /_static/images/search-analytics-demo.png
+   :width: 50%
+   :align: center
+   :alt: Search analytics demo
+
+   How the search analytics page looks.
+
+In **Top queries in the past 30 days**,
+you see all the latest searches ordered by their popularity.
+The list itself is often longer than what meets the eye,
+Scroll downwards on the list itself to see more results.
```

Comment:
This was super annoying to have to write.

But the list has a hidden scrollbar in my browser... Very easy to miss this scroll bar :/